examples/ssm-examples.h

```diff
@@ -133,8 +136,7 @@ int main(int argc, char *argv[]) {
 
   ssm_program_exit();
 
-  for (size_t p = 0; p < allocated_pages; p++)
-    free(pages[p]);
+  ssm_mem_destroy(free_page);
```

Comment:
Right now ssm_mem_destroy doesn't free any memory, right? Is this okay because the pools of pages are statically allocated/don't need to be freed?

Reply:
ssm-runtime's allocator doesn't actually care how the pools are allocated (they could also be dynamically allocated). 

`ssm_mem_destroy` is _supposed_ to handle freeing these pages, but I haven't implemented that just yet.